pet-clinic-web/src/main/resources/messages/messages_pl.properties

```diff
@@ -0,0 +1,8 @@
+welcome=Witamy
+required=jest wymagane
+notFound=nie zostal znaleziony
+duplicate=juz istnieje
+nonNumeric=musi byc liczba
```

Comment:
i believe that UTF-8 to encode characters, how about doing polish translation with a dialectic chars (gęślą jaźń ;) )